paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc

```diff
@@ -42,6 +42,10 @@ void CPUQuantizePlacementPass::ApplyImpl(ir::Graph* graph) const {
       return;
     }
 
+    if (op->Op()->GetAttrIfExists<int>("skip_quant") == 1) {
```

Comment:
The attribute is set to `True`, so I would get its value as `bool`.

Reply:
I have set its value as bool, but raise error. I think Paddle framework change the data type from bool to int.

The saved model as follows.

![image](https://user-images.githubusercontent.com/52520497/104435604-88fbfa80-55c7-11eb-84a3-3dd9b60e9403.png)

Reply:
OK, that's fine then.